test/integration/consul-container/test/basic/connect_service_test.go

```diff
@@ -32,15 +32,13 @@ func TestBasicConnectService(t *testing.T) {
 }
 
 func createCluster(t *testing.T) *libcluster.Cluster {
-	opts := libcluster.BuildOptions{
+	opts := libcluster.ClusterOptions{
 		InjectAutoEncryption:   true,
 		InjectGossipEncryption: true,
-		// TODO: fix the test to not need the service/envoy stack to use :8500
-		AllowHTTPAnyway: true,
```

Comment:
This is the crux of the PR. Existing code will already point the code to the HTTPS port when set correctly. Turning off the plaintext port here.